imports/server/schemas/blob.ts

```diff
@@ -0,0 +1,23 @@
+import * as t from 'io-ts';
+import { Overrides, buildSchema } from '../../lib/schemas/typedSchemas';
+
+export const BlobCodec = t.type({
+  // Base64-encoded blob contents (since Mongo don't play well with null bytes
+  // in string types, and Meteor doesn't play well with Mongo's binary types)
+  value: t.string,
```

Comment:
For the record, if we wanted to do this as a binary, I'm pretty sure we could. EJSON [supports](https://docs.meteor.com/api/ejson.html) `Uint8Array` for binary types, so we just (I think) need to define a `io-ts` codec for it and support converting it to a SimpleSchema in `typedSchemas.ts`. Based on the [io-ts docs](https://github.com/gcanti/io-ts/blob/master/index.md#:~:text=success.-,Example,-A) something like:

```
const uint8Array = new t.Type<Uint8Array>(
  'Uint8Array',
  (input: unknown): input is Uint8Array => input instanceof Uint8Array,
  (input, context) => (input instanceof Uint8Array ? t.success(input) : t.failure(input, context)),
  t.identity
);
```

for the former, and then just add a check for `fieldCodec === uint8Array` somewhere in the conditionals in `buildField`.

I don't have a strong opinion about whether it's _worthwhile_ to try and make this work, but I do think it's at least _possible_.

Reply:
I got this to work!  Thanks for the pointers.